docs/common-bugs.rst

```diff
@@ -322,6 +322,12 @@ Rotated screen
 :Solution: It seems to be a problem with drivers. Reassigning to kernel.
 :Contact: kernel or iio-sensor-proxy
 
+No video output with the MGA G200e graphics card
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Issue: There is no video output with MGA G200e graphic card and a 1920x1080 monitor.
+:Solution: Add ``vga=795`` to the boot options before installation.
```

Comment:
Could you mention also the workaround from https://bugzilla.redhat.com/show_bug.cgi?id=2000537#c1, please?

Reply:
The second workaround seemed not ideal to me (the system will not boot into GUI) but I guess we can mention that so it will be fine.